story-builder/communication/celestial.py

```diff
@@ -3,12 +3,16 @@
 class Celestial(Communication):
 
     def greeting(self):
+        
         return None
 
     def challenge(self):
         return None
 
     def taunt(self):
+        taunt = [ 
+            "Welcome back, Bitch!"
+        ]
         return None
```

Comment:
could you please remove the unnecessary variable `taunt` here? will make the code a bit cleaner if we do it like this:
```python
def taunt(self):
    return [
        "Welcome back, Bitch!",
    ]

```

Reply:
hah, also just noticed you're returning `None` so there was probably some confusion.

We want these methods to return a list of strings that can be randomly selected from.